build.gradle

```diff
@@ -33,7 +33,7 @@ allprojects {
     def awsSdkVersion = '1.11.807'
 
     dependencies {
-        implementation group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: '0.4.3'
+        implementation group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: 'e6b5154'
```

Comment:
Updated nva-commosn to avoid Zalando Problem errors, will set version after the other PR is accepted